virttest/utils_test/libvirt.py

```diff
@@ -953,6 +953,29 @@ def check_actived_pool(pool_name):
     return True
 
 
+def check_vm_state(self, vm_name, state='paused', reason=None, uri=None):
```

Comment:
please remove `self` as it is now not a method associated with class.

Reply:
@balamuruhans Addressed the changes, please review.